Sources/App/Models/Repository.swift

```diff
@@ -36,7 +34,7 @@ final class Repository: Model, Content {
     @Field(key: "forks")
     var forks: Int?
 
-    @OptionalParent(key: "forked_from_id")
+    @OptionalParent(key: "forked_from_id")  // TODO: remove or implement
     var forkedFrom: Repository?
```

Comment:
Do we need this for display? Or would be enough to record that a repo _is_ a fork, without necessarily linking to the original? It would save us a lookup across `reposities` when populating the db and "raceyness" on first ingestion when the fork comes before the parent.